coalib/output/Logging.py

```diff
@@ -33,3 +38,53 @@ def configure_logging():
             }
         }
     })
+
+
+def configure_json_logging():
+    """
+    Configures logging as a JSON objects.
+    Returns a StringIO() that captures the logs as json.
+    """
+    import logging.config
+    import io
```

Comment:
no it's just we usually import everything at the top of the module. Especially in your case I'm not sure what's better, as your function will be normally only called once in an application lifecycle^^

So: keep it :+1: ;)